src/node.js

```diff
@@ -386,6 +404,51 @@
       } while (tickInfo[kLength] !== 0);
     }
 
+    function doNTCallback0(callback) {
+      var threw = true;
+      try {
+        callback();
+        threw = false;
+      } finally {}
+      return threw;
```

Comment:
I just copied from the original function (`_tickCallback`/`_tickDomainCallback`). I can safely delete it I guess? Unless it was originally supposed to catch exceptions?